src/site/content/en/blog/contact-picker/index.md

```diff
@@ -0,0 +1,316 @@
+---
+title: A contact picker for the web
+subhead: The Contact Picker API is an on-demand API that allows users to select entries from their contact list and share limited details of the selected entries with a website. It allows users to share only what they want, when they want, and makes it easier for users to reach and connect with their friends and family.
+authors:
+  - petelepage
+description: Access to the user's contacts has been a feature of native apps since (almost) the dawn of time. The Contact Picker API is an on-demand API  that allows users to select an entry or entries from their contact list and share limited details of the selected contact(s) with a website. It allows users to share only what they want, when they want, and makes it easier for users to reach and connect with their friends and family.
+date: 2019-08-07
+updated: 2019-10-08
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - capabilities
+  - contacts
+  - chrome77
+hero: hero.jpg
+alt: Telephone on yellow background.
+---
+
+
+{% Aside %}
+  The Contact Picker API begins an origin trial in Chrome 77
+  (stable in September) as part of our capabilities project. We'll keep this
+  post updated as the implementation progresses.
+{% endAside %}
+
+## What is the Contact Picker API? {: #what }
+
+<style>
+  #video-demo { max-height: 600px; }
+  figcaption { font-size: small; }
+</style>
+
+<a href="https://storage.googleapis.com/webfundamentals-assets/contact-picker/contact-picker.mp4">
+  <video id="video-demo" loop autoplay muted class="w-figure w-figure--inline-right" style="max-width:50%"
```

Comment:
For images and videos without a solid border it can help to add the `w-screenshot` class. I'd suggest adding it to this element and a few of the other screenshots further down the page.

Reply:
I added it to the `img` vs `figure`, would be nice if I didn't have to add a class for that to happen ;)

Reply:
Would you prefer a shortcode? Images that do have a border don't need it, so it's one of those things that is dependent on the content.

Reply:
No, don't need a short code. Mostly just want to be clear when to use `w-screenshot` and when not.